config.yaml

```diff
@@ -43,7 +43,12 @@ siswrap_service_port: 10700
 
 # The service URL polled in the runfolder sensor:
 runfolder_svc_urls:
-  - http://testarteria1:10800
+  - http://testarteria1:10800/api/1.0/runfolders/pickup
+
+incoming_svc_urls:
+  - 
```

Comment:
yeah that works. During testing I thought it didn't, but the problem was something else and I forgot to change it back.